wikidataQuery.py

```diff
@@ -3,13 +3,17 @@
 import settings
 import traceback
 import json
+import simplejson
+# Installable with 'pip install textblob'
```

Comment:
Please refer to `pip3` instead of `pip`. Afaik `pip` will install the 2.7 python version, which creates unwanted confusion.

Reply:
I referred to pip, because I don't have pip3 command, so I can't tell how it'll react. pip worked for me just fine